components/UploadForm.vue

```diff
@@ -135,14 +135,14 @@ const isDragging = ref(false)
 const epubMetadataList = ref<any[]>([])
 
 const arweaveFee = ref(new BigNumber(0))
-const uploadStatus = ref('')
 const arweaveFeeMap = ref({} as any)
 const arweaveFeeTargetAddress = ref('')
 const sentArweaveTransactionInfo = ref(new Map())
 const balance = ref(new BigNumber(0))
 const isEncryptEBookData = ref(true)
 
-const emit = defineEmits(['arweaveUploaded', 'submit'])
+const emit = defineEmits(['arweaveUploaded', 'submit', 'fileReady'])
+const uploadStatus = defineModel<string>('status')
```

Comment:
Sorry for changing back-and-forth, but since `defineModel` allow parent to modify this variable, maybe we should just stick with `ref` + `watch` + `emit`

Reply:
[♻️ Replace v-model with event emission for upload status](https://github.com/likecoin/nft-book-press/pull/344/commits/e5826132c1ad3902bcb55f052b1790194c6ca740)